pages/profile.js

```diff
@@ -0,0 +1,23 @@
+import Head from "next/head";
+import styled from "styled-components";
+
+export default function Profile() {
+  return (
+    <>
+      <Head>
+        <title>kidsFi - Finance for kids</title>
+      </Head>
+      <Body>
+        <p>Under Construction</p>
```

Comment:
Here you still have the `<Body>` component which might lead to the hydration errors.